examples/astro/astro.config.mjs

```diff
@@ -0,0 +1,16 @@
+import { defineConfig } from "astro/config"
+import react from "@astrojs/react"
+import mdx from "@astrojs/mdx"
+import { remarkCodeHike } from "@code-hike/mdx"
+import theme from "shiki/themes/github-dark.json"
+
+// https://astro.build/config
+export default defineConfig({
+  markdown: {
+    syntaxHighlight: false,
+  },
+  integrations: [
+    react(),
+    mdx({ remarkPlugins: [[remarkCodeHike, { autoImport: false, theme }]] }),
```

Comment:
Setting `autoImport` to `false`, because Astro is unable to find the proper import for `CH` when attempting to hydrate the components otherwise.

I think this may be because `@code-hike` uses the following pattern for auto-importing:
```tsx
import { annotations, Code, Slideshow, CodeSlot } from "@code-hike/mdx/dist/components.cjs.js";

export const CH = {
  annotations,
  Code,
  Slideshow,
  CodeSlot
};
```

Astro [looks at the import statements](https://github.com/withastro/astro/blob/d31e72c3ba8270d1e8d33c533502b3c4c6390a15/packages/astro/src/jsx/babel.ts#L217) when attempting to hydrate a component, so it isn't able to resolve components that use the `CH` namespace here.